src/components/ArrayInput.tsx

```diff
@@ -26,45 +29,59 @@ export const ArrayInput: React.FunctionComponent<IFormtronControl> = ({
   return (
     <Messages path={path}>
       <FieldSet position="relative" variant={variant} legend={schema.title}>
-        {easyArray.items.map((val: any, index: number) => {
-          return (
-            <Flex key={`${index}-${value.length}`}>
-              <Flex flexDirection="column">
-                <Button
-                  type="button"
-                  title="Insert item"
-                  onClick={() => onChange(easyArray.insert(index))}
-                  disabled={disabled}
-                >
-                  <Text color="green">+</Text>
-                </Button>
-                <Button
-                  type="button"
-                  title="Delete item"
-                  onClick={() => onChange(easyArray.remove(index))}
-                  disabled={disabled}
-                >
-                  <Text color="red">x</Text>
-                </Button>
+        {easyArray.items.length === 0 ? (
+          <Text
```

Comment:
This could actually be a `<Button` couldn't it?

Reply:
So... I was wrong about this.
Before:
![image](https://user-images.githubusercontent.com/587740/54208711-cdda1600-44b2-11e9-8f02-9ebd28b25693.png)

After:
![image](https://user-images.githubusercontent.com/587740/54208741-ddf1f580-44b2-11e9-8246-220fc7596441.png)

I can't figure out how to make the icons the correct size. I've tried removing all the padding on the buttons, but for some reason making `button` a `span` results in a different size?

Reply:
ahh hold on let me fix

Reply:
I mean, it definitely makes _sense_ to use Button's, because then we get "disabled" for free instead of the hacky stuff I was doing. But I don't want to sacrifice the good looks. 😦